driver/driver.c

```diff
@@ -89,7 +89,7 @@ DriverEntry(PDRIVER_OBJECT DriverObject,
                                  (PHW_INITIALIZATION_DATA)&WnbdInitData,
                                  NULL);
     if (!NT_SUCCESS(Status)) {
-        WNBD_LOG_ERROR("DriverEntry failure in call to StorPortInitialize. Status: 0x%x", Status);
+        WNBD_LOG_ERROR("DriverEntry failure in call to StorPortInitialize. Status: %x", Status);
```

Comment:
Let's not use "%x", we can stick to "0x%x" or "%#x". Without the "0x" prefix, it's hard to tell if that's a decimal or hexadecimal number if we get something like "Status: 10".

Reply:
Sure, I will change the rest of the occurrences.